modules/accredited_representative_portal/app/services/accredited_representative_portal/representative_user_loader.rb

```diff
@@ -80,5 +80,10 @@ def current_user
 
       @current_user = user
     end
+
+    def verified
+      loa = @current_user.loa.to_h[:current]&.to_i
+      loa == SignIn::Constants::Auth::LOA_THREE
```

Comment:
Same here, `verified` is implied to be `true` (in fact this function is just comparing the constant set above to the same constant, so of course it will always be `true`)